app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -526,6 +526,10 @@ def build_toolbar_hide_button(id)
     return true if @record.kind_of?(OrchestrationStack) && @display == "instances" &&
                    %w(instance_check_compliance instance_compare).include?(id)
 
+    # hide select power management buttons for Openstack::InfraManager
+    return true if %w(host_standby host_shutdown host_reboot host_reset).include?(id) &&
+                   @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
```

Comment:
@PanSpagetka Ok, thanks for the clarification. I moved the code from the basic button into the HostFeatureButton.